### ISSUE TITLE
Fix EPSMixin

### ISSUE THREAD

**Issue by yannikschaelte** [ICB-DCM/pyABC#17]
Make the epsmixin sampler fully object-oriented. 

Also, there seem to be race conditions occuring sometimes and leading to errors, the reasons of which have not been discovered yet.

**Comment by neuralyzer** [comment]
Yeah, I think the stuff using dask is not so very robust. It is not as well tested as the redis stuff. Also, I've made in the past no so good experiences scaling dask to several hundreds cores. Unless the network is very fast, I've experienced lots of timeouts. It should, however, work for smaller core counts.
